src/Microsoft.Extensions.Configuration.UserSecrets/PathHelper.cs

```diff
@@ -3,91 +3,33 @@
 
 using System;
 using System.IO;
-using Microsoft.Extensions.FileProviders;
-using Newtonsoft.Json;
-using Newtonsoft.Json.Linq;
 
 namespace Microsoft.Extensions.Configuration.UserSecrets
 {
+    /// <summary>
+    /// Provides paths for user secrets configuration files.
+    /// </summary>
     public class PathHelper
     {
-        internal const string Secrets_File_Name = "secrets.json";
-        internal const string Config_File_Name = "project.json";
+        internal const string SecretsFileName = "secrets.json";
 
         /// <summary>
-        ///     <para>
-        ///     This method is obsolete and will be removed in a future version. The recommended alternative is <see cref="PathHelper.GetSecretsPathFromSecretsId(string)"/>.
-        ///     </para>
-        ///     <para>
-        ///     Gets the path to the secrets file. If a directory is given, finds the user secrets id in the JSON named 'project.json'.
-        ///     </para>
+        /// <para>
+        /// Returns the path to the JSON file that stores user secrets.
+        /// </para>
+        /// <para>
+        /// This file location uses the current user profile to located the file on disk.
+        /// The intention of obscuring the file location is to avoid accidentally
```

Comment:
It's not really *obscuring* the file location at all, is it? Rather, it's storing it outside of the source control root (which is decidedly *un* -obscure).